apps/api/.env.example

```
@@ -102,6 +102,15 @@ USAGE_EXPORT_BATCH_SIZE=200
 USAGE_EXPORT_TIMEOUT_MS=10000
 USAGE_EXPORT_MAX_ATTEMPTS=10
 
+# One-time credit for a verified OIDC user's default organization. An amount of
+# zero prevents new outbox rows; keeping URL/token set still drains old rows.
+SIGNUP_CREDIT_CENTS=0
+SIGNUP_CREDIT_EXPORT_URL=
+SIGNUP_CREDIT_EXPORT_TOKEN=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SIGNUP_CREDIT_EXPORT_TOKEN key should go before the SIGNUP_CREDIT_EXPORT_URL key

(UnorderedKey)